core/src/main/scala/com.snowplowanalytics.elasticsearch/loader/ElasticsearchSinkApp.scala

```diff
@@ -126,7 +132,10 @@ trait ElasticsearchSinkApp {
     props.setProperty(KinesisConnectorConfiguration.PROP_CONNECTOR_DESTINATION, "elasticsearch")
     props.setProperty(KinesisConnectorConfiguration.PROP_RETRY_LIMIT, "1")
 
-    new KinesisConnectorConfiguration(props, CredentialsLookup.getCredentialsProvider(config.aws.accessKey, config.aws.secretKey))
+    new KinesisConnectorConfiguration(props,
+      CredentialsLookup.getCredentialsProvider(config.aws.accessKey,
+        config.aws.secretKey)
+    ).success
```

Comment:
here you're discarding the result of https://github.com/snowplow/snowplow-elasticsearch-loader/pull/81/files#diff-6a74f84ccb19a31030747933ed804b27R98 and as a result, this function will always return a success.

Whatt you can do is just put every `setPropery` in the `case queue: Kinesis` and build your `KinesisConnectorConfiguration` in the case directly.

Does that make sense?